README.md

```diff
@@ -27,6 +27,7 @@ Versions and dependencies
 | [**2.0**] (*unmaintained*) | ^2.0      | ~2.0.0                   |                |
 | [**1.x**] (*unmaintained*) | ^1.0      | ^2.0                     |                |
 
+
```

Comment:
This extra line added can probably be removed?

Reply:
Sure Line is removed